### PR TITLE
Create artifacts for sharing NuGetGallery.Services

### PR DESCRIPTION
**Pull request by drewgillies**
I did a feasibility spike on getting this project's artifacts into a nupkg for broader use, and got it working using a nupkg stub workaround with built assets (the same approach used for the jobs projects in the same buildstep).

Note that using the `New-ProjectPackage` PS function for this was a dead end, as this project is not SDK-style (the projects in the repo that do use this function are all SDK-style). The msbuild error there is that the `pack` target is not defined, and this target comes built in to SDK-style projects. Converting the Services project into an SDK-style project is also non-trivial as it turns out, so that can possibly be future work.

### CODE REVIEW

**Comment by joelverhagen** [comment]
Legacy csproj should accept pack if you install this package:
https://github.com/NuGet/NuGet.Jobs/blob/f053ae12e6661d7741cb1e096903111c4734f5ef/src/NuGet.Jobs.Common/NuGet.Jobs.Common.csproj#L101-L105 (old version of NuGet.Jobs.Common prior to SDK proj move)
Pack in build.ps1 as proof:
https://github.com/NuGet/NuGet.Jobs/blob/f053ae12e6661d7741cb1e096903111c4734f5ef/build.ps1#L137-L145

File: src/NuGetGallery.Services/NuGetGallery.Services.nuspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <id>NuGetGallery.Services</id>
+    <version>$version$</version>
+    <title>NuGetGallery.Services</title>
+    <authors>.NET Foundation</authors>
+    <owners>.NET Foundation</owners>
+    <description>NuGetGallery.Services</description>
+    <copyright>Copyright .NET Foundation</copyright>
+  </metadata>
+  <files>
+    <file src="bin\$configuration$\**\*.*" target="bin"/>
```
Comment (joelverhagen) [inline src/NuGetGallery.Services/NuGetGallery.Services.nuspec:13]: bin assets are not recognized by NuGet. This package would have no available assets.

**Review by joelverhagen — CHANGES_REQUESTED** [review]
We can use New-ProjectPackage with a dev dependency.

**Comment by drewgillies** [comment]
@joelverhagen Okay, I've addressed these now (thanks again). The `Authors` property was required, and I made it consistent with another project, but I'll go ahead and change it (and the other) to something other than .NET Foundation if desired. Perhaps NuGet. Let me know and I'll reiterate,

**Review by joelverhagen — APPROVED** [review]

**Pull request merged**